server/routes/api/event_logger.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const router = express.Router()
+const logger = require('../../utilities/winston_logger')
+const ErrorLog = require('../../db/models/error_log')
+
+router.post('/log', async (req, res) => {
+  const { severity, data } = req.body
+  res.send(JSON.stringify({ severity, data }))
```

Comment:
I think what makes this vulnerable is you are returning a string of mime-type `text/plain` back instead of `application/json`. Here's a fix I recommend (please test out locally):
```suggestion
  res.json({ severity, data })
```

Reply:
Thank you @paramsiddharth! I tested this solution out locally, and it appears to have resolved the issue.